From 6deb80d22deb2d96d7f3afa643bde848549d43f3 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Tue, 19 Dec 2017 21:02:45 +0800
Subject: [PATCH 3118/5242] MLK-17242-3 arm64: dts: fsl-imx8qm-mek: remove
 properties for power sink

commit  6df755925c664bc07dc9055e46fe7e9579b1bc07 from
https://source.codeaurora.org/external/imx/linux-imx.git

As imx8qm MEK typec port only can support power source on power role,
remove those properties for power sink after we add fixed power sink
settings.

Tested-by: Peter Chen <peter.chen@nxp.com>
Acked-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
index a447de3..ab07cc89 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
@@ -675,10 +675,6 @@
 		ss-sel-gpios = <&gpio4 6 GPIO_ACTIVE_LOW>;
 		reset-gpios = <&gpio4 19 GPIO_ACTIVE_HIGH>;
 		src-pdos = <0x380190c8 0x3803c0c8>;
-		snk-pdos = <0x380190c8>;
-		max-snk-mv = <9000>;
-		max-snk-ma = <1000>;
-		op-snk-mw = <9000>;
 		port-type = "drp";
 		sink-disable;
 		default-role = "source";
-- 
1.7.9.5

